build.xml

```diff
@@ -246,6 +246,8 @@ Copyright (c) 2005, 2017, Oracle and/or its affiliates. All rights reserved.
         <run-jflex dir="${gen.analysis.dir}/javascript" name="JavaScriptXref"/>
         <run-jflex dir="${gen.analysis.dir}/lua" name="LuaSymbolTokenizer"/>
         <run-jflex dir="${gen.analysis.dir}/lua" name="LuaXref"/>
+        <run-jflex dir="${gen.analysis.dir}/posh" name="PoshSymbolTokenizer"/>
```

Comment:
I wouldn't mind if the analyzer would be called 
PowerShell....
this way it is a bit confusing to see what is it for, I hope it's not too posh ;)

Reply:
I can do that. I just was using the vernacular of the PowerShell community.